apps/fyler/src/fyler_hackney_worker.erl

```diff
@@ -0,0 +1,64 @@
+-module(fyler_hackney_worker).
+-behaviour(gen_server).
+-behaviour(poolboy_worker).
+-include("../include/log.hrl").
+-include("fyler.hrl").
+
+-export([start_link/1]).
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2, terminate/2,
+  code_change/3]).
+
+-record(state, {options}).
+
+start_link(Args) ->
+  gen_server:start_link(?MODULE, Args, []).
+
+init(_) ->
+  Options = [{<<"Content-Type">>, <<"application/x-www-form-urlencoded">>}],
+  {ok, #state{options = Options}}.
+
+
+handle_call({response, #task{callback = Callback, file = #file{is_aws = true, bucket = Bucket, target_dir = Dir}},
+  #job_stats{status = success, result_path = Path}}, _From, #state{options = Options} = State) ->
```

Comment:
Line 22 is too long:   #job_stats{status = success, result_path = Path}}, _From, #state{options = Options} = State) ->.